README.md

```diff
@@ -123,21 +130,21 @@ ansible-galaxy collection install -r requirements.yml
 
 The Python module dependencies are not automatically handled by `ansible-galaxy`. To manually install these dependencies, you have the following options:
 
-1. Utilize the `requirements.txt` file to install all required packages:
+1. Install the CrowdStrike FalconPy package directly:
 
     ```terminal
-    pip install -r requirements.txt
+    pip install crowdstrike-falconpy
     ```
 
-2. Alternatively, install the CrowdStrike FalconPy package directly:
+> [!IMPORTANT]
+> If you intend to use Event-Driven Ansible (EDA), the `aiohttp` package should also be installed.
```

Comment:
Make the language less passive.

> If you are building an Event-Driven Ansible (EDA) decision environment, you must also install the `aiohttp` package.

Is that just for event streams, or is there a similar requirement for an AAP execution environment?

Reply:
This is independent of AAP - just an EDA requirement